src/quo2/components/list_items/channel.cljs

```diff
@@ -14,10 +14,11 @@
 
 (defn list-item
   [{:keys [locked? mentions-count unread-messages?
-           muted? is-active-channel? emoji channel-color]
-    :or   {channel-color colors/primary-50}
+           muted? is-active-channel? emoji channel-color
+           default-color]
     :as   props}]
-  (let [standard-props (apply dissoc props custom-props)
+  (let [channel-color  (or channel-color default-color)
```

Comment:
FYI: ClojureScript doesn't allow the default value in `:or` to refer to one of the destructured keys, so I had to do the logic in the `let` macro. I haven't checked if Clojure JVM has this limitation too.